src/main/java/com/example/finance/controller/BudgetController.java

```diff
@@ -0,0 +1,51 @@
+package com.example.finance.controller;
+
+import com.example.finance.business.ApiEndpoints;
+import com.example.finance.model.dto.BudgetDto;
+import com.example.finance.service.BudgetService;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(ApiEndpoints.Endpoints.BUDGET)
+@AllArgsConstructor
+public class BudgetController {
+
+    private final BudgetService budgetService;
+
+    @GetMapping(ApiEndpoints.Endpoints.ID)
+    public ResponseEntity<BudgetDto> getById(@PathVariable UUID id) {
+        return ResponseEntity.ok(budgetService.getById(id));
+    }
+
+    @GetMapping(value = ApiEndpoints.Endpoints.GET_BUDGETS_BY_USER_ID)
+    public ResponseEntity<List<BudgetDto>> getBudgetsByUserId(@PathVariable UUID id) {
+        List<BudgetDto> byUserId = budgetService.getByUserId(id);
```

Comment:
please use correct names for variables - in that case fe. `budgetsByUserId` :) Change it in all methods where it should be done